cmd/sdk-server/main.go

```diff
@@ -85,10 +89,22 @@ func main() {
 	defer cancel()
 
 	if ctlConf.IsLocal {
-		err = registerLocal(grpcServer, ctlConf)
+		localSdk, err := registerLocal(grpcServer, ctlConf)
```

Comment:
Random thought - should this be integrated with the local mode, or should it be it's own --test mode?

(and maybe be it's own separate implementation, rather than mixed in with the local implementation?)

Reply:
Yes, I will try to separate test mode but them looks identical for now.

Reply:
If it doesn't work out, it doesn't work out - was just wondering if it would make things cleaner.

Reply:
I think test mode should have a bit different constructor but other parts would look the same. So I am going to reuse Local SDK server code, not to duplicate code.

Reply:
Sounds good! We can also split at a later date if we decide there is need.